VideoLocker/res/values/colors.xml

```diff
@@ -94,7 +94,6 @@
     <color name="video_details_background_transparent">#CC1F2124</color>
     <color name="delete_panel_bg">#3E4247</color>
     <color name="empty_list_text">#c8c8c8</color>
-    <color name="button_text_disabled">@color/edx_grayscale_neutral_base</color>

```

Comment:
This should be converted to a theme attribute, in order to be easily customizable.